### DISCUSSION TITLE
Determine resources (people/projects/funds) needed to operate and sustain

### DISCUSSION THREAD

**Discussion by gusaus** [communitycontributorgroup/ccg#9] [Strategy and Operations]
_11/22/23 Update: We're in the process of transitioning (from Community Contributor Group) in preparation for officially launching as  [FOSS Contributors](http://www.fosscontributors.org/). Will be updating this description as we reassess our team and needs._

We need to identify roles, responsibilities, and commitments needed for our [open startup](https://github.com/orgs/fosscontributors/discussions/1) to function and grow. Not a community organizer myself, I can only reference roles and examples of other communities I'm part of (or aware of) in hope that other team members with experience can help define (and ideally assume an ongoing role).

## FOSS Contributors 
Still in the process of building the team; for now we'll just list the folks interested in assuming some type of role

### Core Team
- [Gus Austin](https://opencollective.com/gusaus) / [FOSS Contributors](https://opencollective.com/fosscontributors) / [OpenProducer](https://opencollective.com/openproducer) / [FundOSS](https://opencollective.com/fundoss)  / [Drupal Contribution](https://opencollective.com/drupal-contribution) - Outreach, onboarding, community liaison, partnerships / focusing on Drupal and WordPress projects in Music, Community Media, & Education verticals
- [Tony Zeoli](https://opencollective.com/tony-zeoli) / [FOSS Contributors](https://opencollective.com/fosscontributors) / [OpenProducer](https://opencollective.com/openproducer) - Strategy, product, partnerships / focusing on WordPress projects in Broadcasting, Community Media, & Publishing verticals
- [Miguel Hernandez](https://opencollective.com/techivist) / [FOSS Contributors](https://opencollective.com/fosscontributors) / [OpenProducer](https://opencollective.com/openproducer) - Outreach, onboarding, community liaison, partnerships / focusing on Drupal and WordPress projects in Music, Community Media, & Education verticals
- [Anita Ihuman](https://opencollective.com/anita-ihuman) / [FOSS Contributors](https://opencollective.com/fosscontributors) / [Drupal Contribution](https://opencollective.com/drupal-contribution) - Dev Advocate, Technical Writing, Documentation  
- [Victory Brown](https://opencollective.com/victory-brown) / [FOSS Contributors](https://opencollective.com/fosscontributors) / [Drupal Contribution](https://opencollective.com/drupal-contribution) - Community manager, researcher and writer.

### Contributors/Advisors
- [Schalk Neethling](https://opencollective.com/schalkneethling) / [FOSS Contributors](https://opencollective.com/fosscontributors) / [Mechanical Ink](https://opencollective.com/mechanical-ink) / [The Mycelium Network](https://opencollective.com/the-mycelium-network) - Frontend Engineer, Open Web Standards, and Accessibility Evangelist. Designer. Musician.
- [Shane Curcuru](https://opencollective.com/shanecurcuru) / [FOSS Contributors](https://opencollective.com/fosscontributors)  - Director @ ASF; independent FOSS volunteer

### Team Needs
We need skilled/experienced people interested/able to work across multiple projects and programs at a [competitive rate](https://github.com/orgs/fosscontributors/discussions/5) or salary (down the line). There are most likely enough projects/programs in our network of orgs/collectives to justify steady part or full-time work for folks assuming these roles (which we need to detail and post). 

- Project Manager/Coordinator
- Proposal Writer
- Development Director/Fundraising Manager
- Program Manager

### Planning discussions
Threads or discussions about priority projects and programs:

- [WordPress](https://github.com/orgs/fosscontributors/discussions/9#discussioncomment-7542710)
- [Drupal](https://github.com/orgs/fosscontributors/discussions/9#discussioncomment-9438177)
- [FundOSS](https://github.com/orgs/fosscontributors/discussions/9#discussioncomment-7542708)
- #2

**Comment by Victorybrown** [comment]
Awesome work @gusaus This is a well put together issue. How soon do we intend to do this if these are the positions we're looking to fill? If the responsibilities appear to be closely related, do you think some roles can be combined. Most of the sources you cited in relation to the problem appear to be larger organisations with contributors and possibly funding.

**Reply by gusaus** [reply]
Thanks for chiming in @Victorybrown!

Couple things to note -

the roles outlined are intended to get us started - and absolutely can be modified, combined, or expanded (this is really where folks like yourself and actually most everybody that has joined this repo has some experience and could help define)
we're talking about the 'Open Collective community team' where there currently is no collective associated with it (but something we should probably request https://github.com/communitycontributorgroup/opencollective/issues/6); ideally our time will be funded through our [Community Contributor Group collective](https://opencollective.com/community-contributor-group); in that case we'd be contributing as a partner org; in the same way other team members might end up contributing on behalf of their company or employer
while Open Collective does not have capacity to enable its own community - they do have team members (like Richard) that spend part time on aspects of the community. So ideally, once we're a bit more defined - more folks from Open Collective will play an ongoing role.
Hopefully, that provides more clarity and not confusion.

Main thing to keep in mind is we're starting organically so we all can help create roles that seem interesting and fulfilling!

**Comment by Anita-ihuman** [comment]
> Thanks for chiming in @Victorybrown!
> 
> Couple things to note -
> 
> * the roles outlined are intended to get us started - and absolutely can be modified, combined, or expanded (this is really where folks like yourself and actually most everybody that has joined this repo has some experience and could help define)
> * we're talking about the 'Open Collective community team' where there currently is no collective associated with it (but something we should probably request [Create 'Community' collective #6](https://github.com/communitycontributorgroup/opencollective/issues/6)); ideally our time will be funded through our [Community Contributor Group collective](https://opencollective.com/community-contributor-group); in that case we'd be contributing as a partner org; in the same way other team members might end up contributing on behalf of their company or employer
> * while Open Collective does not have capacity to enable its own community - they do have team members (like Richard) that spend part time on aspects of the community. So ideally, once we're a bit more defined - more folks from Open Collective will play an ongoing role.
> 
> Hopefully, that provides more clarity and not confusion.
> 
> Main thing to keep in mind is we're starting organically so we all can help create roles that seem interesting and fulfilling!

Great work @gusaus, this is well put together. 
and clearly explained.
So, some interested individuals may have to  wear more than one hat(Just as you are doing) until the respective roles are occupied.

**Reply by gusaus** [reply]
I think the first step should be for everybody involved to create their own roles and time commitment that works for them.

For me, I'm wearing too many hats and spending more time than I would like to (at least without a budget). @Anita-ihuman @Victorybrown I'd recommend crafting roles that you're comfortable with and estimating your availability based on whether or not there's a budget.

I think documentation, community, and project management are our biggest needs, and it seems like you both have the first two covered. I'm aware it hasn't been easy to dive in, but I feel like we're close to being able to make contributions to multiple projects (for instance Open Collective, Drupal, and WordPress) that have similar needs.

Would love to hear from @ShaneCurcuru and @schalkneethling on what would be a realistic roles/time commitment (even if its just a couple hours here and there advising).

I'm just winging it - so any input from those with experience building community teams is a huge help!

**Comment by ShaneCurcuru** [comment]
I think the first steps are to slightly better define the goals, and create a single homepage somewhere that briefly describes what we hope to become (apparently an OC hosted community) and what we want to do (provide content and support/ideas/mentoring to other OC hosted open source groups).
Then to get enough people to show up here to say "I can _volunteer_ for X amount of Y kind of thing", or to say "I can do X once we get a _paid_ OC host thing going".

I can _volunteer_ a small amount of time for community bootstrapping, and hope to later volunteer time at building truly re-usable content for open-source related communities.  My interest is both in content/guides/etc. that help any FOSS community, and also finding ways to encourage and show FOSS communities why they might want to come to OC.  This is part of the larger sustainability movement I'm trying to help define and grow.

**Comment by gusaus** [comment]
@ShaneCurcuru Following up on a couple of your comments and questions from our [planning discussion](https://opencollective.slack.com/archives/C94E046SY/p1680869158630379) in Slack.

>  I keep forgetting that becoming a “collective” means the potential for actual funding/payment for collective-directed work (i.e. not just all volunteers all the time).

The collective is important since it's why the [company](https://opencollective.com/) and community exists.

Most people here (aside from the relatively small [Open Collective team](https://docs.opencollective.com/help/about/team)) are users/customers with collectives. The majority are small 1-2 person open source projects with similar needs (contributors and funds).

There are outliers like [Sustain](https://sustainoss.org/) ([#sustainoss](#sustainoss), [#sustain-links](https://opencollective.slack.com/archives/C02T1KXS997)) which consists of folks from Open Collective, OSPOs and other orgs/groups focusing on sustaining open source. It was through that group (specifically this [Sustain Together google doc](https://docs.google.com/document/d/1b4dYI8MZiGSCnsXaqxvrM0mtdKmhj4DnuHxjs3kByig/edit?usp=sharing)) where I connected to @Anita-ihuman and @Victorybrown after realizing we had several similar interests, goals, and ideas!  

But for the most part, these groups are siloed because Open Collective has little capacity beyond serving their customers and sustaining their platform team.

This leaves a need/opportunity for us to enable community working groups! https://github.com/communitycontributorgroup/opencollective/issues/2

**Comment by schalkneethling** [comment]
@gusaus @Victorybrown @ShaneCurcuru @Anita-ihuman, it almost feels like we need a mission statement. From what I could gather in this thread, here is my attempt at it.

> Our mission at the Open Collective community contributor group is to empower open-source projects, maintainers, and communities by providing the resources, support, and mentorship necessary for their success, health, and inclusivity.
>
> As a team made up of volunteers and paid core team members who have firsthand experience with Open Collective, we are uniquely positioned to offer guidance on best practices and help others understand the benefits of this platform.
>
> We strive to foster a welcoming and collaborative environment where new contributors can enhance our existing skills and work alongside us to improve the open-source community. Through the provision of content such as guides and project templates, as well as ongoing support and mentorship, we aim to help other Open Collective hosted open source groups thrive and reach their full potential.

Let me know your thoughts.

**Comment by gusaus** [comment]
@schalkneethling Considering it's still difficult to catch up and understand everything (especially when I'm driving without a map), this is pretty close to the goals and priorities we're outlining in https://github.com/communitycontributorgroup/opencollective/issues/10

**Comment by schalkneethling** [comment]
> @schalkneethling Considering it's still difficult to catch up and understand everything (especially when I'm driving without a map), this is pretty close to the goals and priorities we're outlining in [communitycontributorgroup/opencollective#10](https://github.com/communitycontributorgroup/opencollective/issues/10)

Should we close this one and move the conversation to https://github.com/communitycontributorgroup/opencollective/issues/10? Perhaps the idea with this one is to eventually list the roles and the people who will fill these roles?

**Reply by gusaus** [reply]
@schalkneethling The reason I copied certain comments to https://github.com/communitycontributorgroup/opencollective/issues/10 and hid them here was to create seperation between the Open Collective (open source platform and community) and the [FOSS Contributors](https://github.com/orgs/communitycontributorgroup/discussions/1) (a nonprofit service we're setting up to provide support and services to Open Collective and other open source projects and communities).

I previously transferred this issue to [the primary repo](https://github.com/communitycontributorgroup/ccg) for FOSS Contributors, but I'm not sure I informed the group!

The description and 'visible' comments are now on topic, but I'll need to bring 'this' group up to speed on how we'll be leveraging [our collective](https://opencollective.com/fosscontributors) to pay for our time and/or credit our sponsoring orgs.

I'll try to elaborate before the work week begins!

**Comment by gusaus** [comment]
@ShaneCurcuru @schalkneethling @Anita-ihuman @Victorybrown I've added the following to the issue description but wanted to leave as a comment to emphasize we need a few more skilled/available folks before we can move everything forward. 

There's no shortage of paid work or funded projects in the ecosystem. The problem is we don't 'seem to' have anyone on the team with skills and/or availability to create effective plans and proposals. Without that, it's difficult for anyone to make a regular, ongoing time commitment. 

We're not talking about much upfront time for people with these skills. And their time would be paid on the back end. 

But how do we find them?

> We need skilled/experienced people interested/able to work across multiple projects and programs at a competitive rate (when budget becomes available) or salary (down the line). There are most likely enough projects/programs in our network of orgs/collectives to justify steady part or full-time work for folks assuming these roles (which we need to detail and post).
> 
> - Proposal Writer
> - Project Manager/Coordinator
> - Development Director/Fundraising Manager
> - Program Manager
>

**Comment by gusaus** [comment]
Quick followup on my last comment and related discussions in Slack... I wanted to make sure everybody understands that nobody is being asked or expected to contribute recurring time unless it it is paid (unfortunately on the back end) or you're working on behalf of your company or employer (who would receive in-kind sponsor credit). 

My sense is/was that @ShaneCurcuru and @schalkneethling might be able to spend sporadic time here and there mainly advising and collaborating as a partner org or community ([The Mycelium Network](https://opencollective.com/the-mycelium-network), [Mechanical Ink](https://opencollective.com/mechanical-ink), [MDN Web Docs](https://developer.mozilla.org/en-US/), [Apache](https://www.apache.org/)) on projects/programs where there's overlap and mutual benefit.  

@Anita-ihuman @Victorybrown and me _could_ potentially allocate regular, recurring time between multiple projects, programs, groups, and communities if the collectives associated with them had budgets to draw from. 

While I've been emphasizing the need for proposal writers and project managers, we (the folks with availability) could also spend x amount of time contributing to a particular community (such as Open Collective and Drupal) if there was a budget available in associated collectives. 

I'm going to revisit some conversations with potential recurring sponsors next week.

Also would be great to know what everybody's interest/potential availably (for paid, sponsored work) is looking like in May.

**Reply by gusaus** [reply]
Circling back to this particular thread (a little over a year later) as our ongoing participation in high profile/impact programs in Drupal and WordPress communities 'could' be converted into paid opportunities in the near term. As both are leveraging Open Collective (platform and community) our active roles in the project and community could/should also lead to paid contributions on this side.

Of course this sounds very similar to the opportunities we were discussing in comments from last year (many I hid because they're now outdated or repetitive), anything related to direct funding in global open source ecosystems (like WordPress and Drupal) is slow going and complicated. 

But the fact is we're currently helping drive those efforts as part of high profile/impact teams and working groups (read through referenced threads in the description https://github.com/orgs/fosscontributors/discussions/9#discussion-5839637 for more context on priority projects and programs).

With regards to our team, @tekNorah (check https://github.com/tekNorah for more on all she brings to the table) recently joined the collaboration in the Drupal [#innovation](https://drupal.slack.com/archives/C05K610S0F8) Slack channel and has subsequently joined collaboration channels in WordPress and Open Collective.
 
@Victorybrown and @Anita-ihuman have shown interest in helping increase the WordPress/Drupal presence in Africa...and we've had preliminary discussions with leadership on both sides about how we can help (via enabling mentored contribution programs, hackathons, and fundraisers).

@techivist, someone I met and collaborated with during the early days of Drupal in Los Angeles, has been onboard to help increase the presence in LA, California, and Latin America.   

Even through I've talked with most of the folks referenced, it would be better if everybody could comment in this thread about current interest/availability and desired contribution roles.

Keeping in mind again that while contribution to projects like Drupal and WordPress is opt-in and unsponsored, we're setting ourselves up to prove that direct funding (for our time/efforts) can increase contributions while enabling a much more diverse community.

**Comment by gusaus** [comment]
Realizing it's been awhile since anyone (myself included) updated/added/removed their availability, I'll summarize the call to action I just left in this [Slack discussion](https://opencollective.slack.com/archives/G4NDU96PR/p1698256241564929) as a motivator. 

I mentioned we're tentatively planning the next [FundOSS](https://opencollective.com/fundoss) (an [org](https://github.com/FUNDOSS) I/we now help run) for sometime between [GivingTuesday](https://www.givingtuesday.org/) (November 28) and end of year. 

In addition to FundOSS, we'd help run rounds for any of the following (that can sponsor their own round) similar to how [Gitcoin](https://grants.gitcoin.co/ ) runs their program rounds (but at a much smaller, test the waters scale). 

- [ ] [OpenProducer](https://github.com/orgs/OpenProducer/discussions/31)
- [ ] [WordPress](https://docs.google.com/document/d/1CtVJZdx1oNer9FeCIROlx-IiVh1XkrDKuycdR75phfs/edit#bookmark=id.kvgrol3n9y5x)
- [ ] [Drupal](https://docs.google.com/document/d/13Cqc4bhlBZ5q4bqdEhjvq9EnAOXwjpJSrxwg4JBpb6Y/edit?usp=sharing) 

I then went on to mention the FundOSS platform isn't quite functional (see https://github.com/orgs/FUNDOSS/discussions/44 for fairly recent context) but we've been talking to the original partners/key members about working with them to bring it back (if they could  product own/manage and ideally help fund and/or post a bounty to entice skilled folks that can complete in a timely fashion). 

At the same time we've been talking to Gitcoin about extending their [Grants Stack](https://www.gitcoin.co/grants-stack), repurposing [organizer resources](https://manager-toolkit.super.site/) while exploring additional ways to partner and pool resources.

If we're unable to secure funds and fill talent gaps next week, we'll most likely put off scheduling the next round until early next year. But once we start, we'll most likely run a round every quarter - this is on top of the events and services we've been planning with community contributors and ecosystem partners.

I'll try to keep everybody appraised in related issues and discussions. At the same time I'll keep nudging everybody in Slack  (and here) to provide a quick status update about roles, interests, and availability.

Looking at mine in [the issue description](https://github.com/communitycontributorgroup/ccg/issues/3#issue-1668643921), it looks like I'll need to add [FundOSS](https://opencollective.com/fundoss), [Drupal Sustainability](https://www.drupal.org/project/sustainability), and [WordPress Sustainability](https://make.wordpress.org/sustainability/) to my list of organizations/collectives and teams. All of those have similar contributor needs and paid opportunities (in the short term).

So a lot to do for anyone available and interested in getting paid (and potentially employed) to make impactful open source contributions!

**Reply by gusaus** [reply]
So crib notes on what's happened since this last comment - 

- I'm now an admin/owner on https://opencollective.com/fundoss and https://github.com/FUNDOSS
- We have permission to use [the budget](https://opencollective.com/fundoss#category-BUDGET) in the latter to begin the process of reviving and expanding FundOSS
- We're trying to enroll FundOSS in the upcoming/current [Gitcoin Grants](https://grants.gitcoin.co/) round (to raise additional funding)
- We're solidifying and exploring a variety of other partnerships in the [Open Collective ecosystem](https://discover.opencollective.com/)
- We still need to determine which @communitycontributorgroup/developers and @communitycontributorgroup/community are available (as independent contractors or partner orgs) for the FundOSS and related work being described in this thread.

If it sounds like theres an interesting opportunity, please chime in!

**Reply by gusaus** [reply]
Circling back here to reference a couple comments in the [FundOSS revival discussion](https://github.com/orgs/FUNDOSS/discussions/44) that should provide context around progress and status.

#### December 2023
Put everything on hold at the end of last year https://github.com/orgs/FUNDOSS/discussions/44#discussioncomment-7912604
> Quick update to say we tabled our push to have a functional FundOSS platform until admins/partners of the pilot had a chance to revisit and reasses their role, commitment, investment, and benefits in future Democratic/Quadratic Funding rounds.
> 
> While some of the new admins and partners have made progress identifying participants and sponsors (Drupal and WordPress seem very interested in ongoing rounds), we've decided to table the idea of FundOSS or Simplegrants platforms as a stopgap and see if extending/contributing to https://www.gitcoin.co/grants-stack (and reconstituting our partnership with Gitcoin) could be an option in early(ish) 2024.
> 
> Still would require some input from @BenJam @owocki and/or whoever they recommend should be driving partnership and product discussions on behalf of Open Collective/OSC and Gitcoin.
> 
> Ideally we'd be able to reconnect after the holidays and see if/how we can go back to the source and create something complimentary to https://impact.gitcoin.co/

#### May 2024
Discovered we might soon have a platform we can leverage and contribute to  https://github.com/orgs/FUNDOSS/discussions/44#discussioncomment-9289615

> Dusting off this thread after realizing @owocki and Gitcoin 🙌 have restarted development of https://github.com/gitcoinco/simplegrants.xyz 📣🤘
> 
> Skim back through this thread and discussion for context around how we were planning to use (or at least evaluating) as a replacement for https://github.com/FUNDOSS/fundoss
> 
> I haven't talked to Kevin recently (not sure if follow team members in this discussion have) but it seems like contributing to SimpleGrants (and revisiting/reconstituting the partnership with [Gitcoin](https://www.gitcoin.co/)) would enable us to get FundOSS (and some of these [other projects/programs](https://github.com/orgs/FUNDOSS/discussions/44#discussioncomment-7503340)) rolling again 🎬
> 
> Would love to hear thoughts from some of the other folks involved.


I've yet to reconnect with Kevin or other FundOSS team members (will be following up in an email this week), but the fact that FundOSS is listed as an example of _Some of the projects funded via the Simple Grants methods_ (on https://www.simplegrants.xyz/) would seem to indicate the open source platform (https://github.com/FUNDOSS/simplegrants.xyz) to could not only power ongoing FundOSS rounds (see https://github.com/FUNDOSS/fundoss/issues/47 for examples), but also ongoing [Drupal](https://www.drupal.org/project/innovation_ideas/issues/3376257) and [WordPress](https://github.com/WordPress/sustainability/issues/22) rounds.

#### Next steps
Assuming I'll be able to reconnect with fellow FundOSS team members next week, we should soon have a better sense of status of the platform, resources needed to manage ongoing rounds, and where to centralize collaboration. 

For the time being lets keep using https://github.com/FUNDOSS/fundoss and [this Canvas](https://opencollective.slack.com/docs/T0HSL38JD/F05H1N51QR2) (should be able to access when logged into Open Collective Slack) for the latter. 


Does that make sense?

**Reply by gusaus** [reply]
Here's the email I sent to Kevin (https://github.com/owocki) and a few other fellow FundOSS admins/owners yesterday (Ben and Pia are also key Open Collective [team](https://docs.opencollective.com/help/about/team) members).

> Kevin, 
> 
> I was updating forked repos last week and realized https://github.com/gitcoinco/simplegrants.xyz was being actively developed by https://github.com/gitcoinco! I [commented](https://github.com/orgs/FUNDOSS/discussions/44#discussioncomment-9289615) in the related FundOSS discussion but felt compelled to circle back here after realizing 'Get started' inquiries on https://www.simplegrants.xyz/ were going to you (when I first started drafting this email last week).
> 
> After reading over the site, skimming GitHub [issues](https://github.com/gitcoinco/simplegrants.xyz/issues), and playing around with https://demo.simplegrants.xyz/ it seems like a white label solution (fork?) of https://github.com/gitcoinco/simplegrants.xyz would be the logical choice to power platforms for FundOSS, QF for [Open Collective cities](https://blog.opencollective.com/cities-are-open-collectives/)/regions ([Downtown Stimulus](https://medium.com/@katiebrenjohnson/downtown-stimulus-program-review-e89f67e29f4f) looks like a great model!) and potentially stand-alone platforms for Drupal, WordPress, and other large projects/communities with their own ecosystem.
> 
> That being said, I wanted to make sure that would be possible before asking about status, contributor/resource needs, and how we might be able to revisit, reconstitute, and extend the partnerships assembled for the FundOSS pilot.
> 
> Looping in Ben and Pia as I've been holding off on bugging any of my fellow FundOSS admins until https://opencollective.com/fosscontributors (and partners) had capacity to rebuild/contribute to a platform in addition to managing/marketing ongoing rounds.
> 
> I feel like we're there and would love to reconnect and discuss next steps (if you think we're at a point to do so).
> 
> Congrats on what looked like another successful GG round. Hoping we'll also be able to participate and contribute to the next one!

**Comment by gusaus** [comment]
With regards to [the vision](https://github.com/orgs/communitycontributorgroup/discussions/2) of the band when it got together in April, we're trying to remove one of the blockers by forming a supergroup with the original visionary. https://github.com/communitycontributorgroup/opencollective/issues/6#issuecomment-1793532648

If that happens - we're going to be much closer to having the 'financial' resources needed to begin regular rehearsals to prepare for several world tours beginning next year.

Hope you like the music analogy - pretty accurate actually considering it's [already a focus area](https://github.com/orgs/OpenProducer/discussions/31) for one of our partners!

**Comment by gusaus** [comment]
@Anita-ihuman @Victorybrown @techivist As mentioned in this [Slack thread](https://opencollective.slack.com/archives/G4NDU96PR/p1698256241564929), I'm going to do my best to spend 5 hours this week (the time I've pledged on https://profiles.wordpress.org/gusa/) focusing on items we've already identified as priorities on the WordPress [Sustainability Team roadmap](https://github.com/orgs/WordPress/projects/134). If you're interested/able to commit a similar amount of time, we can connect in WordPress Slack after you've done the following:

- [ ] Create/finish your WordPress.org profile
   - [ ] [Login](https://login.wordpress.org/) or [Register](https://login.wordpress.org/register)
   - [ ] [Pledge](https://wordpress.org/five-for-the-future/for-individuals/) a realistic amount of Hours per week and select  "Community Team" and "Sustainability Team"
- [ ]  [Join Make WordPress Slack](https://us.wordcamp.org/2022/joining-make-wordpress-slack/) and the following channels (feel free to join others that seem interesting)
  - [ ] [#sustainability](https://wordpress.slack.com/archives/C03JNV77Y57)
  - [ ] [#community-team](https://wordpress.slack.com/archives/C037W5S7X)
- [ ] Catch up on https://make.wordpress.org/sustainability/ by skimming through the following:
  - [ ] Skim through [Chat Summaries](https://make.wordpress.org/sustainability/category/chat-summaries/)
  - [ ] Posts related to the [Team](https://make.wordpress.org/sustainability/category/team/)
  - [ ] [Roadmap items](https://github.com/orgs/WordPress/projects/134/views/1)
  - [ ] [Admin tasks ](https://github.com/WordPress/sustainability/issues?q=is%3Aissue+is%3Aopen+label%3Aadmin)
  - [ ] [Contribution Goals and Ideas (GitHub discussions)](https://github.com/WordPress/sustainability/discussions/categories/contribution-goals-and-ideas) 
- [ ] DM [gusaus](https://wordpress.slack.com/archives/DKKKWDP0X) and I'll invite you to our private Canvas next time I'm online 

Considering the Sustainability Team is still in the process of getting up and running, it's a great opportunity to make connections with other contributors and Teams interested in contributing to the projects and programs we've already prioritized on the Sustainability roadmap (and in this issue)!

**Reply by gusaus** [reply]
@Anita-ihuman @Victorybrown @techivist With regards to the previous comment, The WordPress marketing team actually made getting up and running there easier by launching the following Contributor Onboarding Flow:

 [Contribute](https://make.wordpress.org/contribute/) - Find the team that aligns with your interests
 [Join](https://make.wordpress.org/contribute/join/) - signup for your [WordPress.org](http://wordpress.org/) and Slack accounts
If at all possible, could you complete those steps by Friday so we can async participate in the [Sustainability Team Slack meeting](https://make.wordpress.org/sustainability/2023/11/08/sustainability-chat-agenda-november-10-2023/)? That would be a great opportunity to claim https://github.com/WordPress/sustainability/issues/2, [roadmap items](https://github.com/orgs/WordPress/projects/134), [admin tasks](https://github.com/WordPress/sustainability/issues?q=is%3Aissue+is%3Aopen+label%3Aadmin), and connect with the team and the larger community.

I'll be happy to show you around and make intros - assuming you're ready to roll!

**Reply by gusaus** [reply]
@Anita-ihuman @Victorybrown @techivist Per recent Slack updates and discussions, we in the beginning stages of moving the following project forward on the WordPress Sustainability Team [Roadmap](https://github.com/orgs/WordPress/projects/134):

- [ ] [Enable/organize/contribute to NextGen and other Events related to Sustainability goals and focus areas](https://github.com/WordPress/sustainability/issues/12)
- [ ] [Refine and extend Five for the Future](https://github.com/WordPress/sustainability/issues/16)
- [ ] [Explore options for funding community projects, teams, working groups, and contributors](https://github.com/WordPress/sustainability/issues/15)
- [ ] [Help connect and streamline collaboration between Make Teams and community contributors](https://github.com/WordPress/sustainability/issues/14)

In addition to priority projects, there 'might' be some [admin/team building](https://github.com/WordPress/sustainability/issues?q=is%3Aissue+is%3Aopen+label%3Aadmin) tasks you'd like to help with.

Leave a comment here and/or followup with me in WordPress Slack so we can plot a course forward!

**Reply by gusaus** [reply]
I left a couple comments in the [Five for the Future](https://github.com/WordPress/sustainability/issues/16) project after listening to this [SustainWP](https://sustainwp.com/) podcast [episode](https://sustainwp.com/podcast/episode-5-wordpress-cms-how-can-we-make-it-more-sustainable/) where several Sustainability Team (reps and members) delve into solving similar problems and solutions. 

We all can contribute and benefit. I'm ready to make intros and recommendations on specific roles and tasks. Just need to say the word!

**Comment by gusaus** [comment]
While team members have been involved with multiple Drupal projects and programs leveraging Open Collective since the start, we've recently been focusing on the [Innovation initiative](https://www.drupal.org/innovation/) as a catch-all (to pool contributors and funding).

In addition to this [issue tracker](https://www.drupal.org/project/innovation_ideas), 'most' collaboration is centered in [#innovation](https://drupal.slack.com/archives/C05K610S0F8) (Drupal Slack), so contributors should make sure to skim [the canvas](https://drupal.slack.com/canvas/C05K610S0F8) and discussions in the channel (referenced below).

* [Innovation initiative planning discussion](https://drupal.slack.com/archives/C05K610S0F8/p1709928905866299)
* [Innovation Hub proposal (and planning)](https://drupal.slack.com/archives/C05K610S0F8/p1712531491244829)
* [Growth and innovation in Africa (and other regions)](https://drupal.slack.com/archives/C05K610S0F8/p1711990168513269)
* [Inspiration and examples for a Drupal Innovation Fund](https://drupal.slack.com/archives/C05K610S0F8/p1710787583596559)
* [Contributing (and enabling contributions) to Starshot](https://drupal.slack.com/archives/C05K610S0F8/p1715048024216649)

Chime in here with questions about specific contributor needs and benefits.

**Comment by gusaus** [comment]
Spinning up a thread to focus on finalizing and launching [this initial offering](https://opencollective.slack.com/docs/T0HSL38JD/F06SP7907PT) of paid (customer facing) services focusing on [Newspack](https://newspack.com/). The action items and meeting notes in the canvas should provide background and context

Not reflected are the partnership discussions we've been having with the [Newspack team](https://newspack.com/meet-the-team/) over the past few years. I referenced those in [our working doc](https://docs.google.com/document/d/1i2ZVRNOCUGm0F7qTvPjZ8P38SnjqBvaGvSosfoFOqSg/edit?usp=sharing) (which everybody should now be able to view)...

......at the very bottom you'll see the most recent interactions, including my proposed next steps

> We're hoping to finalize and launch service offerings for OpenProducer and agency partners by the end of the month.
> As our ideal again is to pool resources, contribute, and collaborate with Newspack/Automattic, any feedback on what we're proposing (assuming it makes sense) would help us position our value prop and partner offerings so they are complimentary.
> I'll follow up again when things are finalized and would be happy to answer questions, provide additional detail/clarity, or even jump on a call beforehand.

Due to the unforeseen circumstances, we've yet to launch or circle back to Newspack. At the same time, the Newspack team and partnership/collaboration opportunities in their ecosystem have grown (this [LinkedIn post](https://www.linkedin.com/feed/update/urn:li:activity:7203461901144506369/) from their founder and head of product is just the latest example).

And considering Newspack is part of [Automattic](https://automattic.com/), our contributions to [hackathons](https://github.com/WordPress/sustainability/discussions/40) and [fundraisers](https://github.com/WordPress/sustainability/issues/22) as part of the [WordPress Sustainability team](https://make.wordpress.org/sustainability/) already gives us an entry point for collaboration, partnership, and ideally funding considering most leaders in the WordPress community are [sponsored](https://wordpress.org/five-for-the-future/pledge/automattic/) by Automattic.

That said, I'd like to see if we can finalize, launch, and provide the service mix outlined in [this section](https://docs.google.com/document/d/1i2ZVRNOCUGm0F7qTvPjZ8P38SnjqBvaGvSosfoFOqSg/edit#heading=h.gh5zgzqnjyh7).

The other outcome will be the mechanism to contribute and provide related services around Drupal and their new [Starshot Initiative](https://www.drupal.org/about/starshot).

**Reply by gusaus** [reply]
I added [these details](https://automattic.com/for-agencies/program-incentives/) about Automattic's new [Agency partnership program](https://automattic.com/for-agencies/) to our [services doc](https://docs.google.com/document/d/1i2ZVRNOCUGm0F7qTvPjZ8P38SnjqBvaGvSosfoFOqSg/edit#heading=h.oj2owan4obx)  as it seems like there could be multiple ways to generate additional revenue / cut costs for our [platform support offering](https://docs.google.com/document/d/1i2ZVRNOCUGm0F7qTvPjZ8P38SnjqBvaGvSosfoFOqSg/edit#bookmark=id.pklto9c8tktn)

**Reply by gusaus** [reply]
Also reached out to our fiscal sponsor [Open Source Collective](https://oscollective.org/) to make sure it would be would be ok to enroll [OpenProducer](https://opencollective.com/openproducer) and/or [FOSS Contributors](https://opencollective.com/fosscontributors) (effectively on their behalf).

Will provide an update here after they review https://automattic.com/for-agencies/platform-agreement/ and circle back to me.
